pkgs/development/python-modules/indexed_zstd/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, isPy3k
+, cython
+, zstd
+}:
+
+buildPythonPackage rec {
+  pname = "indexed_zstd";
+  version = "1.6.0";
+
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "89c09eaebbfa8a10634934b816cc3baa1a00e616bcc9e81f31546220e853bef6";
+  };
+
+  nativeBuildInputs = [ cython zstd ];
+
+  buildInputs = [ zstd ];
```

This runs and builds for me
```suggestion
  buildInputs = [ zstd pkgs.ztd.dev ];
```

```shell
$ /home/dylan/src/nixpkgs/result/bin/ratarmount --version
ratarmount 0.14.0
ratarmountcore 0.6.0

System Software:

Python 3.10.12
FUSE 2.9.9
libsqlite3 3.42.0

Compression Backends:

indexed_bzip2 1.4.1
rapidgzip 0.10.1
indexed_gzip 1.8.5
xz 0.4.0
lzmaffi 0.3.0
indexed_zstd 1.1.3
rarfile 4.0
```